tests/certification/bindings/azure/cosmosdb/components/masterkey/cosmosdb.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: azure-cosmosdb-binding
+  namespace: default
+spec:
+  type: bindings.azure.cosmosdb
+  version: v1
+  metadata:
+  - name: url
+    secretKeyRef:
+        name: AzureCosmosDBUrl
+        key:  AzureCosmosDBUrl
+  - name: database
+    secretKeyRef:
+        name: AzureCosmosDB
+        key:  AzureCosmosDB
+  - name: collection
+    secretKeyRef:
+        name: AzureCosmosDBCollection
+        key:  AzureCosmosDBCollection
```

Comment:
Nit: Do these need to be secrets? I'm fine keeping them that way, but if they weren't it's a bit easier to know where this is pointing. Same question applies to the other cosmosdb yamls.

Reply:
You just discovered how our conformance test infrastructure works :)

All constants, names and secrets must be provisioned by the `setup-azure-conf-test.sh` script. The values it generates are then exported to KeyVault.

The Conformance Test and Certification Test workflows loads just the required values from KeyVault and injects those into Environment Variables.

To load these Environment Variables I am using the local env var secret store. But to refer to those "environment secrets" I then must specify in the manner seen here.

Reply:
I used the same trick for enabling Azure components in E2E tests. This is a nice way to pass values that change per run.